Makefile

```diff
@@ -464,6 +464,21 @@ lint-metrics-md: metrics-docs ## Check if metrics documentation is up to date.
 		false; \
 	fi
 
+.PHONY: validate
+validate: ## Convenience target running linters, formatters and generators across the codebase.
```

Comment:
any chance we can run some of those in parallel?

Reply:
good point, yeah, I'll update :+1: